include/deal.II/hp/fe_collection.h

```diff
@@ -948,8 +948,8 @@ namespace hp
 
     unsigned int max = 0;
     for (unsigned int i = 0; i < finite_elements.size(); ++i)
-      if (finite_elements[i]->n_dofs_per_face() > max)
-        max = finite_elements[i]->n_dofs_per_face();
+      if (finite_elements[i]->max_dofs_per_face() > max)
+        max = finite_elements[i]->max_dofs_per_face();
```

Comment:
```suggestion
      max = std::max(max, finite_elements[i]->max_dofs_per_face());
```

Reply:
I have applied your suggestion to all the other functions in the file too.

I also saw these `if(val > max) max = val;` and didn't like it either. I wanted to be smart and use map-reduce, but it turned out that std algorithms don't have that... That would be one line of code :(